File: TS_JT-Reader/TS_JT-Reader_Vis-Studio/TS_JT-Reader/DeeringLookupEntry.ts
module C_sharp_JT_Reader.Codecs {
    export class DeeringLookupEntry {
        cosTheta: number;
        sinTheta: number;
        cosPsi: number;
        sinPsi: number;
        constructor(cosTheta: number, sinTheta: number, cosPsi: number, sinPsi: number) {
            this.cosTheta = cosTheta;
            this.sinTheta = sinTheta;
            this.cosPsi = cosPsi;
            this.sinPsi = sinPsi;
        }
        public getCosTheta(): number {
            return this.cosTheta;
        }
        public getSinTheta(): number {
            return this.sinTheta;
        }
        public getCosPsi(): number {
            return this.cosPsi;
        }
        public getSinPsi(): number {
            return this.sinPsi;
        }
    }
}

File: TS_JT-Reader/C#_JT-Reader_Vis-Studio/C sharp JT-Reader/Codecs/DeeringLookupEntry.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace C_sharp_JT_Reader.Codecs
{
    public class DeeringLookupEntry
    {
        double cosTheta;
        double sinTheta;
        double cosPsi;
        double sinPsi;

        public DeeringLookupEntry(double cosTheta, double sinTheta, double cosPsi, double sinPsi)
        {
            this.cosTheta = cosTheta;
            this.sinTheta = sinTheta;
            this.cosPsi = cosPsi;
            this.sinPsi = sinPsi;
        }

        public double getCosTheta()
        {
            return cosTheta;
        }

        public double getSinTheta()
        {
            return sinTheta;
        }

        public double getCosPsi()
        {
            return cosPsi;
        }

        public double getSinPsi()
        {
            return sinPsi;
        }

    }
}